Spring for GraphQL and React (#1444)
oktadeveloper/okta-blog: _source/_posts/2023-07-27-spring-boot-graphql-react.md

---
layout: blog_post
title: "How to Build a GraphQL API with Spring Boot"
author: jimena-garbarino
by: contractor
communities: [security,java,javascript]

> description: "A step-by-step guide for building a secured GraphQL API with Spring Boot and Auth0 authentication on React"

Reviewer: ```suggestion
description: "A step-by-step guide to building a secure GraphQL API with Spring Boot, React, and Auth0."
```
